src/mlpack/methods/det/dtree_impl.hpp

```diff
@@ -15,9 +15,6 @@
 #include <stack>
 #include <vector>
 
-using namespace mlpack;
-using namespace det;
-
 namespace details
 {
```

Comment:
Actually I think the intention here was to have this be in `mlpack::det::details`, so maybe it is worth doing this here?

```suggestion
namespace mlpack {
namespace det {
namespace details {
```

That would probably also mean you could revert some of the changes that add `mlpack::` or `mlpack::det::` below.